ntbs-service/Pages/Notifications/Create.cshtml.cs

```diff
@@ -18,12 +18,6 @@ public CreateModel(INotificationRepository repository)
         public async Task<IActionResult> OnGetAsync()
         {
             var notification = new Notification();
```

Comment:
Why are we getting rid of this? I think this will break something when we try to do the updates in NotificationService, as context.Entry(notification....) will be null and we're trying to call CurrentValues on it?

Reply:
I have initalised properties in Notification class constructor

Reply:
Ah great!